lessons/Projects/project-design-a-survey/langs/en-us/pages/survey-hacking.adoc

```diff
@@ -0,0 +1,39 @@
+= Survey Hacking
+
+Exchange surveys with another group and test the limits of their survey by trying to enter as much "dirty" data as you can! 
+
+@n Which survey are you testing out?
+
+@fitb{}{}
+
+@n For each dirty data example, write down what you submitted and what suggestion you would have to help the other group guard against it.
+
+[cols="1a,^4a,^6a", options="header"]
+|===
+| Question Number
+| Dirty Data Entered
+| Suggestion for Improving the Survey Question
```

Comment:
Why not use `.FillVerticalSpace` here?

Reply:
I think you must've made this page before `.FillVerticalSpace` was around. I'll fix.